public_include/fty/message-bus.h

```diff
@@ -0,0 +1,93 @@
+/*  =========================================================================
```

Comment:
This malamute wrapper is nice. It's also in fyi-automatic-groups and also in etn-ipm2-network. Maybe we could have it as a "common" library... @perrettecl